src/client/common/utils/multiStepInput.ts

```diff
@@ -87,20 +90,33 @@ export class MultiStepInput<S> implements IMultiStepInput<S> {
         activeItem,
         placeholder,
         buttons,
-        shouldResume
+        shouldResume,
+        matchOnDescription,
+        matchOnDetail,
+        acceptFilterBoxTextAsSelection
     }: P): Promise<MultiStepInputQuickPicResponseType<T, P>> {
         const disposables: Disposable[] = [];
         try {
             return await new Promise<MultiStepInputQuickPicResponseType<T, P>>((resolve, reject) => {
                 const input = this.shell.createQuickPick<T>();
-                input.title = title;
+                if (title !== undefined) {
+                    input.title = title;
+                }
```

Comment:
```suggestion
                input.title = title;
```
Any reason why it cant be this? `title` is `undefined` by default anyway right.

Reply:
Right, I didn't notice that title is optional.